test/controlplane/suite/testcase.go

```diff
@@ -123,7 +124,7 @@ func (cpt *ControlPlaneTest) SetupEnvironment(modConfig func(*agentOption.Daemon
 	agentOption.Config.Debug = true
 
 	operatorOption.Config.Populate(operatorCmd.Vp)
-	operatorOption.Config.SkipCRDCreation = true
+	cpt.operatorHandle.hive.Viper().Set(client.SkipCRDCreation, true)
```

Comment:
Here the `cpt.operatorHandle` is still `nil`, since the operator has not yet been initialized. This line should be moved in `StartOperator`, just after the initialization of the operator handle.